py/server/deephaven/table.py

```diff
@@ -279,6 +285,7 @@ def move_columns_up(self, cols: Union[str, Sequence[str]]) -> Table:
         except Exception as e:
             raise DHError(e, "table move_columns_up operation failed.") from e
 
+    @auto_locking_op
```

Comment:
It seems a little weird that `rename_columns` needs locking, but `drop_columns` or `move_columns*` don't.